### PR TITLE
#164139643 Implement password reset feature

### PR DESCRIPTION
**Pull request by youngestdj**
#### What does this PR do?
This PR consumes the password reset endpoints 
#### Description of tasks to be completed?
* Add `requestpasswordreset` page
* Add `resetpassword` page
#### How can this be manually tested?
* Navigate to the `/requestpasswordreset` and `resetpassword` pages in your browser.
#### Screenshots
image
image

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/164139643

### CODE REVIEW

**Review by olajide1234 — DISMISSED** [review]

**Review by Dubby20 — DISMISSED** [review]

**Comment by 9jaswag** [comment]
@youngestdj have you addressed the feedback raised by Lekan? You also have merge conflicts here.

**Comment by youngestdj** [comment]
@9jaswag I have implemented feedback. I'll fix merge conflicts shortly.

**Review by olajide1234 — DISMISSED** [review]

**Review by kleva-j — DISMISSED** [review]

File: src/views/passwordReset/RequestPasswordReset.jsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+  const updateInput = e => setState({
+    ...state,
+    email: e.target.value,
+    error: null,
+    message: null
+  });
+
+  const completeRequest = async (e) => {
+    e.preventDefault();
```
Comment (9jaswag) [inline src/views/passwordReset/RequestPasswordReset.jsx]: @youngestdj what is `e`? You should make your params readable. Use `event` instead....and please pass this to the team

**Review by 9jaswag — CHANGES_REQUESTED** [review]

**Comment by 9jaswag** [comment]
@youngestdj I also did not get a screenshare for this feature.

**Comment by 9jaswag** [comment]
@youngestdj ?

**Comment by youngestdj** [comment]
@9jaswag I'm fixing merge conflicts in staging, I'll send a screenshare soon.

**Review by codinger41 — DISMISSED** [review]

**Review by kleva-j — DISMISSED** [review]

File: src/views/passwordReset/RequestPasswordReset.jsx (hunk 1 above)
Comment (youngestdj) [reply src/views/passwordReset/RequestPasswordReset.jsx]: I have fixed this @9jaswag

File: src/routes/Index.jsx [hunk 2]
```diff
@@ -4,12 +4,17 @@ import NotFound from '../components/404';
...
+import RequestPasswordReset from '../views/passwordReset/RequestPasswordReset';
+import ResetPassword from '../views/passwordReset/ResetPassword';
 
 const Routes = () => (
   <Switch>
+    <Route path="/" exact component={LandingPage} />
     <Route path="/login" exact component={Login} />
     <Route path="/articles/:slug" component={Article} />
-    <Route path="/" exact component={LandingPage} />
+    <Route path="/articles/:slug" component={Article} />
```
Comment (9jaswag) [inline src/routes/Index.jsx]: @youngestdj why is this route declared twice?

Comment (youngestdj) [reply src/routes/Index.jsx]: Fixed. @9jaswag

**Review by 9jaswag — APPROVED** [review]

**Pull request merged**